schema/crdb/9.0.0/up1.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TYPE IF NOT EXISTS omicron.public.ip_pool_resource_type AS ENUM (
+    'fleet',
+    'silo'
```

Comment:
```suggestion
    'silo',
    'fleet'
```

The ordering here was wrong which could break things. Tricky because I assume this was automatically generated. Seems like it changing the order is a bug?

Reply:
This isn't automatically generated, it's manually written to produce the same thing as what's in dbinit.sql. The bug is me!

Reply:
Actually, though... the fact that it's not caught by the schema tests might be a bad thing. We've recently made the tests a little less sensitive (https://github.com/oxidecomputer/omicron/pull/4288), but I would not have expected that to affect enum order.